api/tables.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Classes for interfacing with specific database tables.
+ * Original design by Zarel in https://github.com/Zarel/telepic/blob/master/server/db.ts, redone by Mia
+ * @author mia-pi-git
+ */
+import {Config} from './config-loader';
+import {DatabaseTable} from './database';
+
+import type {LadderEntry} from './ladder';
+import type {PreparedReplay, ReplayData} from './replays';
+import type {UserInfo} from './user';
+
+export const users = new DatabaseTable<UserInfo>('ntbb_users', 'userid');
+
+export const ladder = new DatabaseTable<LadderEntry>(
+	'ntbb_ladder', 'entryid', Config.ladderdb
+);
```

Comment:
Isn't the `ntbb_` prefix supposed to be configurable? If not, it should be made configurable.